internal-api/src/main/kotlin/com/makarytskyi/internalapi/topic/KafkaTopic.kt

```diff
@@ -2,8 +2,10 @@ package com.makarytskyi.internalapi.topic
 
 object KafkaTopic {
     private const val REPAIRING_BASE = "repairing"
+    private const val ORDER_BASE = "order"
     private const val USER_BASE = "user"
 
+    const val ORDER_CREATE = "$ORDER_BASE.create"
```

Comment:
maybe add new object inside to group topic due to domain
so then you could write `KafkaTopic.Order.CREATE`